### ISSUE TITLE
Adding a new group fails with error message, but group is actually added to LDAP

### ISSUE THREAD

**Issue by awesome-manuel** [KircheNeuenburg/ldaporg#34]
We use groups with objectClass "groupofnames". This app adds a new group with objectClass "posixGroup". Nextcloud LDAP provider is configured to only consider groups with objectClass "groupofnames", so maybe this causes the error message when creating a new group.

**Comment by alexhor** [comment]
yes, currently only posixGroup is supported as objectClass for groups.
I will add groupofnames for you.
Might take a few days though.
